test/blocks/global-navigation/test-utilities.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable no-promise-executor-return */
+/* eslint-disable import/prefer-default-export */
+import sinon, { stub } from 'sinon';
+import { setViewport } from '@web/test-runner-commands';
+import initGnav from '../../../libs/blocks/global-navigation/global-navigation.js';
+import {
+  getLocale,
+  setConfig,
+  loadStyle,
+} from '../../../libs/utils/utils.js';
+import defaultPlaceholders from './mocks/placeholders.js';
+import defaultProfile from './mocks/profile.js';
+import largeMenuMock from './mocks/large-menu.plain.js';
+import globalNavigationMock from './mocks/global-navigation.plain.js';
+import { isElementVisible } from '../../../libs/blocks/global-navigation/utilities/keyboard/utils.js';
+
+export { isElementVisible };
+
+export const selectors = {
```

Comment:
Is there any way we could leverage the selectors we already have in `keyboard/utils.js`?